src/Patterns/Patterns.php

```diff
@@ -0,0 +1,79 @@
+<?php
+
+namespace AstraticBlocks\Patterns;
+
+use AstraticBlocks\Patterns\Parser;
+use WP_Block_Patterns_Registry;
+use WP_Block_Pattern_Categories_Registry;
+
+class Patterns
+{
+  private string $dir;
+  private array $patterns;
+  private array $allowedFiles;
+  private Parser $parser;
+  private const ABSL_PATTERN_MAIN_CATEGORY = 'Astratic Patterns';
```

Comment:
I believe it should be A **_SB_** L but this is a "class" const anyway so no need for prefixing imho 😅